nautobot/dcim/views.py

```diff
@@ -235,15 +257,6 @@ def get_extra_context(self, request, instance):
         }
 
 
-class LocationTypeEditView(generic.ObjectEditView):
-    queryset = LocationType.objects.all()
-    model_form = forms.LocationTypeForm
-
-
-class LocationTypeDeleteView(generic.ObjectDeleteView):
-    queryset = LocationType.objects.all()
-
-
 class LocationTypeBulkImportView(generic.BulkImportView):  # 3.0 TODO: remove, unused
```

Comment:
Shouldn't `LocationTypeBulkImportView` and `LocationTypeBulkDeleteView` be removed as well?